src/braket/circuits/result_types.py

```diff
@@ -44,6 +44,19 @@ def __init__(self):
     def to_ir(self) -> ir.StateVector:
         return ir.StateVector.construct()
 
+    @classmethod
+    def from_ir(cls, ir_result) -> StateVector:
```

Comment:
Could we please add the type annotations to the parameters for these `from_ir` methods across all the files? 
(`def from_ir(cls, ir_result: ir.StateVector)`)

Reply:
Yes, added type annotations to all `ir_result` arguments.